src/parser/Literal.h

```diff
@@ -92,5 +92,12 @@ class Literal {
   static Literal literalWithoutQuotes(
       std::string_view rdfContentWithoutQuotes,
       std::optional<std::variant<Iri, std::string>> descriptor = std::nullopt);
+
+  // Set the substring of the literal by erasing the part between the
+  // end of the prefix and the trailing " from content_.
```

Comment:
The comment is confusing, 
erase everything but the substring in the range `[start, start+length)` from the inner content. Note that the `start` position does not count the leading quotes, so the first character after the quote has index 0. Throws if either `start` or `start + length` is out of bounds.